internal/worker/dbaccessor/tracker.go

```diff
@@ -132,6 +132,18 @@ func newTrackedDBWorker(
 		return nil, errors.Trace(err)
 	}
 
+	// Set the maximum number of idle and open connections to 2 (default is 0).
+	// Not setting these values can lead to a large number of open connections
+	// being created and not closed, which can lead to unbonded connections
+	// being left open.
+	//
+	// TODO (stickupkid): We will want to move this to runtime.GOMAXPROCS(0)
+	// when we have a better understanding of the impact of this change.
+	db.SetMaxIdleConns(2)
```

Comment:
The doc says the default is 2
`By default an sql.DB keeps two idle connections at any given moment`

Reply:
I'll update the comment to state that both should be the same, that way when we do change them it'll be obvious.